index.html

```diff
@@ -9,7 +9,8 @@
 	<script src="/socket.io/socket.io.js"></script>
 	<script src="https://code.jquery.com/jquery-2.2.3.min.js" integrity="sha256-a23g1Nt4dtEYOj7bR+vTu7+T8VP13humZFBJNIYoEJo=" crossorigin="anonymous"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.12.0/lodash.min.js"></script>
-	<script src="/js/main.js"></script>
+	<script type="text/javascript"  src="bower_components/underscore/underscore-min.js"></script>
```

Comment:
You're adding `bower` and `underscore` as dependencies, but `lodash` is right there on line 11 and it does all `underscore` does, and is better at it :)